src/components/datagrid/DataGrid.js

```diff
@@ -121,14 +127,31 @@ export class DataGridComponent extends FormioComponents {
     });
 
     // Add the add button if not disabled.
-    if (!this.shouldDisable) {
-      let tr = this.ce('tr');
-      let td = this.ce('td', {
-        colspan: (this.component.components.length + 1)
-      });
-      td.appendChild(this.addButton());
-      tr.appendChild(td);
-      this.tbody.appendChild(tr);
+    if (!this.shouldDisable && (
+      !this.component.addAnotherPosition ||
+      this.component.addAnotherPosition === "bottom" ||
+      this.component.addAnotherPosition === "both"
+      )){
+        let tr = this.ce('tr');
+        let td = this.ce('td', {
+          colspan: (this.component.components.length + 1)
+        });
+        td.appendChild(this.addButton());
+        tr.appendChild(td);
+        this.tbody.appendChild(tr);
+
+      // if(
```

Comment:
Do not comment out code. If you don't need it, then just remove it. GIT helps us keep track of things removed and added.